nostr-client-plus/src/bin/aggregator.rs

```diff
@@ -171,11 +174,13 @@ async fn run() -> Result<()> {
                                 match validate_result {
                                     Err(msg) => {
                                         tracing::error!("Validate error {}", msg.to_string());
+                                        FAILED_JOBS.inc();
```

Comment:
can we separate these two? so we can have per error reason counters

Reply:
I dont see the necessary, there are actually about 5 fail cases in the verify result, seperate into 5 is too over complicated

Reply:
@wkq7740 I merged the diff but let's separate these two so we have better visibility